modules/management-console-agent/src/main/java/org/apache/ignite/internal/agent/dto/action/query/QueryDetailMetrics.java

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright 2020 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.agent.dto.action.query;
+
+import org.apache.ignite.internal.util.typedef.internal.S;
+
+/**
+ * Query detail metrics.
+ */
+public class QueryDetailMetrics {
```

Comment:
Is this good idea to use same name as interface org.apache.ignite.cache.query.QueryDetailMetrics in core module?

Reply:
I think yes, we can change name anytime, because we don't depends on anything.